pretrained_models/alexnet.py

```diff
@@ -0,0 +1,187 @@
+"""
+A Keras Implementation of the GoogleNet (InceptionV1) from
+https://github.com/heuritech/convnets-keras/blob/master/convnetskeras/convnets.py
```

Comment:
This comment is incorrect.

Reply:
You are right. I wrote the name of architecture wrong. 

```suggestion
A Keras Implementation for Alexnet from
https://github.com/heuritech/convnets-keras/blob/master/convnetskeras/convnets.py
```